docs/language/operators.tex

```diff
@@ -0,0 +1,73 @@
+In our language we support many operations on numbers and booleans.
+
+For numbers we support:
+\begin{itemize}
+	\item $+$
+	\item $-$
+	\item $*$
+	\item $/$
+	\item $^\wedge$
+	\item $\%$
+	\item $SQRT$
+	\item $>$
+	\item $<$
+	\item $<=$
+	\item $>=$
+\end{itemize}
```

Comment:
In de PDF resulteert dit in een nogal lange opsomming welke ook gewoon met komma's achter elkaar kan worden gezet om niet zo veel ruimte te verspillen als nu

Lijkt mij mooier, maar is een optionele aanpassing